lib/numbers.md

```diff
@@ -5,20 +5,20 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+In Ruby there are Fixnum (or Integer) numbers as well as Float numbers. 
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Common operations you would perform on numbers would include your usual PEMDAS math operations as well as the modulus operation. Comparisons would include "greater than >", "less than <" as well "equals to =="
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+With the "+" operation on a number, ruby interprets that as the addition operation. So "2+4" ruby would return "6". However, with Strings the "+" operation servers as a concatenator, thus literally appending one string to the other.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+A mathematical operation can be used on a string, however, not with the expected results. For example the string "20" multiplied by 5 would return 2020202020. In order to make the previous example return the expected answer, you would need to convert the string to an integer using the ".to_i" command.
```

Comment:
If you had a string `"20"` and wanted to divide it by 3, would you use `"20".to_i` to make it work? Think about (or investigate) what happens if you do use `to_i` in this situation and report back your thoughts.